app/tests/system/pageObjects/androidActions.js

```diff
@@ -0,0 +1,82 @@
+/*
+ Put Android-specific actions in this page object.
+
+ Do not perform verifications here. Verifications and assertions should go
+ either in the test script or, for commonly used assertions, placed in an
+ assertions object in tests/system/assertions.
+*/
+
+var selectors = {
+    // Define selectors here
+
+    // This xpath is the example, please update the xpath.
+    HAMBURGER_MENU_BUTTON_XPATH: '//android.widget.LinearLayout[1]/android.widget.FrameLayout[1]/android.support.v4.widget.DrawerLayout[1]/android.widget.LinearLayout[1]/android.widget.LinearLayout[1]/android.widget.RelativeLayout[1]/android.widget.FrameLayout[1]/android.widget.ImageView[1]'
+};
+
+var AndroidAction = function(browser) {
+    this.browser = browser;
+    this.selectors = selectors;
+};
+
+AndroidAction.prototype.clickHamburgerMenuButton = function() {
+
+    var self = this;
+
+    this.browser
+        .contexts(function(result) {
+            self.browser
+                .useXpath()
+                // set context to NATIVE_APP
+                .setContext(result.value[0])
+                .currentContext(function(ctx) {
+                    self.browser.log('The context after switching ' + ctx.value);
+                })
+                .waitForElementVisible(selectors.HAMBURGER_MENU_BUTTON_XPATH)
+                .log('Clicking Hamburger Menu button')
+                .click(selectors.HAMBURGER_MENU_BUTTON_XPATH)
+                //Waiting for the page load completely
+                .pause(5000)
```

Comment:
On mobile, waiting for page loading is tricky and necessary. We tried to use .waitUntilMobified(), but this is not enough, and sometime it doesn't work. If the page is not loading properly, the case will be failed. We haven't find a best way to handle it.